### DISCUSSION TITLE
What are your world's currencies?

### DISCUSSION THREAD

**Discussion by cstby** [cstby/foundryvtt-world-currency-5e#30] [Q&A]
We want to know about your world's custom currencies! Your feedback will help us prioritize features and bug fixes for this module.

Are you using this module to implement the Silver Standard? Are you just hiding electrum? Are you using currencies unique to your fictional world? Something else entirely? We would love to know!

**Comment by kmuld** [comment]
I personally use this module for the Silver Standard.

**Comment by themalkav** [comment]
Currently I run, Copper as .01, Silver as .25, Gold as 1.00 and Mithril as 100.00 and Adman as 1,000 also I have a Clockwork city that uses Cog variants, and EP doesnt exist however I have Stellarite as well sadly I cant get this program of yours to work, and nothing out there lets me edit currency... I miss using it :(, so my players just write the notes down. I wanted to make other coins, yet without editing, I wasnt able to for adding more currency types mimic my Wild West Goblins with Guns campaign which I've been recording on Rumble and YouTube. Just stopped by in hopes to find out if you ever planned to update this or something and let you know, I've yet to find a replacement for it.

<
<img width="1024" height="1024" alt="B7 Coin Silver" src="https://github.com/user-attachments/assets/d2cfafbd-2a4a-4b1f-b230-47c8e4c9c9e3" />
<img width="1024" height="1024" alt="B7 Coin Stellarite" src="https://github.com/user-attachments/assets/fc56fe5d-b672-4b91-8fe3-26d03659f557" />
<img width="1024" height="1024" alt="B7 Coin Mithril" src="https://github.com/user-attachments/assets/06b95b1b-1b56-4dad-988a-a7dee8a79dfa" />
<img width="1024" height="1024" alt="B7 Coin Admantine" src="https://github.com/user-attachments/assets/fb9da556-436f-4219-b989-0b152fc60638" />
<img width="1024" height="1024" alt="B7 Coin Copper" src="https://github.com/user-attachments/assets/e3f7d8ed-d675-4bb5-92be-e9f1b4aedc39" />
<img width="1024" height="1024" alt="B7 Coin Gold" src="https://github.com/user-attachments/assets/bc10ce24-fe8a-482a-b5dc-34e766efe963" />
